jericho/jericho.py

```diff
@@ -601,11 +541,13 @@ def copy(self):
         env.set_state(state)
         return env
 
+
```

Comment:
PEP8: methods of a class should be separated by one newline. There are a few unneeded newlines below as well.